src/solc_0.8/asset/ERC1155ERC721.sol

```diff
@@ -54,34 +53,47 @@ contract ERC1155ERC721 is WithSuperOperators, IERC1155, IERC721 {
 
     mapping(address => address) private _creatorship; // creatorship transfer
 
+    // @review Bouncer
     mapping(address => bool) private _bouncers; // the contracts allowed to mint
-    mapping(address => bool) private _metaTransactionContracts; // native meta-transaction support
-
+    mapping(address => bool) private _metaTransactionProcessors; // meta-transaction support
+    // @review Bouncer ... Deprecate?
     address private _bouncerAdmin;
```

Comment:
we could  also make it immutable, it works great with proxy, you configure them in the constructor but they are hardocded in the implemention
but last time we checked openzeppelin do not support that and will generate an error

Reply:
WIll look into this and see where oz is at with adding support. I know they've been working on it.

Reply:
This feature is not supported yet :(
https://github.com/OpenZeppelin/openzeppelin-upgrades/issues/312